tests/hikari/test_guilds.py

```diff
@@ -558,6 +754,94 @@ def test_make_banner_url_when_no_hash(self, model):
         model.banner_hash = None
         assert model.make_banner_url(ext="png", size=2048) is None
 
+    @pytest.mark.asyncio
+    async def test_fetch_roles(self, model):
+        model.app.rest.fetch_roles = mock.AsyncMock()
+
+        roles = await model.fetch_roles()
+
+        model.app.rest.fetch_roles.assert_awaited_once_with(123)
+        assert roles is model.app.rest.fetch_roles.return_value
+
+    @pytest.mark.asyncio
+    async def test_fetch_owner(self, model):
+        model.app.rest.fetch_member = mock.AsyncMock()
+
+        owner = await model.fetch_owner()
+
+        model.app.rest.fetch_member.assert_awaited_once_with(123, 1111)
+        assert owner is model.app.rest.fetch_member.return_value
+
+    @pytest.mark.asyncio
+    async def test_fetch_widget_channel(self, model):
+        model.app.rest.fetch_channel = mock.AsyncMock()
+
+        widget_channel = await model.fetch_widget_channel()
+
+        assert widget_channel is model.app.rest.fetch_channel.return_value
```

Comment:
You can combine these lines. I'm going to guess this was done as you saw some test cases storing the return value in `returned` before checking. This is done due to black formatting being hard to read in asserts which are really long

```suggestion
        assert await model.fetch_widget_channel() is model.app.rest.fetch_channel.return_value
```

Reply:
Yea, I saw the other tests and with it. I will change these